src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        Ball secondBall = Lottery.getRandomBall();
```

Comment:
Don't use static methods in your solution. Instead of calling `Lottery.getRandomBall()` as a static method, create an instance of the `Lottery` class and call the method on that instance.